examples/Transcoder/Server/Server.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="..\Proto\greet.proto" GrpcServices="Server" Link="Protos\greet.proto" />
+
+    <PackageReference Include="Grpc.AspNetCore" Version="$(GrpcDotNetPackageVersion)" />
```

Comment:
Minor nit: Since this is a sample maybe it makes sense to hardcode the version for folks who are copy-pasting?

Reply:
Hard coding with so many samples was a nightmare to update so many references. The README.md in the examples directory talks about what you need to do if a sample is copied out of the repo.